setup.py

```diff
@@ -62,11 +62,70 @@
     },
     entry_points=dict(
      console_scripts=[
-     # <hack>
-     ] + ['{}=spinalcordtoolbox.compat.launcher:main'.format(os.path.splitext(x)[0]) for x in os.listdir("scripts") if x.endswith(".py")] + [
-     # </hack>
-      # TODO add proper command-line entry points from refactored code
-      #'sct_deepseg_gm=spinalcordtoolbox.deepseg_gm.__main__:main',
-     ],
+        '{}=spinalcordtoolbox.compat.launcher:main'.format(x) for x in \
+        [
+            'isct_check_detection',
+            'isct_convert_binary_to_trilinear',
+            'isct_minc2volume-viewer',
+            'isct_test_ants',
+            'isct_warpmovie_generator',
+            'msct_multiatlas_seg',
+            'sct_analyze_lesion',
+            'sct_analyze_texture',
+            'sct_apply_transfo',
+            'sct_check_dependencies',
+            'sct_compute_ernst_angle',
+            'sct_compute_hausdorff_distance',
+            'sct_compute_mscc',
+            'sct_compute_mtr',
+            'sct_compute_mtsat',
+            'sct_compute_snr',
+            'sct_concat_transfo',
+            'sct_convert',
+            'sct_create_mask',
+            'sct_crop_image',
+            'sct_deepseg',
+            'sct_deepseg_gm',
+            'sct_deepseg_lesion',
+            'sct_deepseg_sc',
+            'sct_denoising_onlm',
+            'sct_detect_pmj',
+            'sct_dice_coefficient',
+            'sct_dmri_compute_bvalue',
+            'sct_dmri_compute_dti',
+            'sct_dmri_concat_b0_and_dwi',
+            'sct_dmri_concat_bvals',
+            'sct_dmri_concat_bvecs',
+            'sct_dmri_create_noisemask',
+            'sct_dmri_display_bvecs',
+            'sct_dmri_moco',
+            'sct_dmri_separate_b0_and_dwi',
+            'sct_dmri_transpose_bvecs',
+            'sct_download_data',
+            'sct_extract_metric',
+            'sct_flatten_sagittal',
+            'sct_fmri_compute_tsnr',
+            'sct_fmri_moco',
+            'sct_get_centerline',
+            'sct_image',
+            'sct_label_utils',
+            'sct_label_vertebrae',
+            'sct_maths',
+            'sct_merge_images',
+            'sct_pipeline',
+            'sct_pipeline_makefig',
```

Comment:
we could probably remove them (replaced by sct_run_batch)

Reply:
Delete reference and script. Added `sct_run_batch` reference.